core/frontend/src/tile/GltfReader.ts

```diff
@@ -2172,7 +2172,17 @@ export abstract class GltfReader {
     const offset = bv.byteOffset ?? 0;
     buf = buf.subarray(offset, offset + bv.byteLength);
 
-    const mesh = await loader.parse(buf, { modules: { draco3d } }); // NB: `options` argument declared optional but will produce exception if not supplied. Regardless, we are specifying our own bundled draco3d module to avoid CDN requests.
+    const mesh = await loader.parse(buf, {
+      draco: {
+        decoderType: "wasm",
+      },
+      modules: {
+        "draco_wasm_wrapper.js": `${IModelApp.publicPath}draco/draco_wasm_wrapper.js`,
+        "draco_decoder.wasm": `${IModelApp.publicPath}draco/draco_decoder.wasm`,
+      },
+      worker: false,
+      useLocalLibraries: true,
+    }); // NB: `options` argument declared optional but will produce exception if not supplied. Regardless, we are specifying our own bundled draco3d module to avoid CDN requests.
```

Comment:
update the comment here, were not bundling draco3d anymore

Reply:
updated